tests/integration-tests/tests/update/test_update/test_dynamic_file_systems_update/storage-stack.json

```diff
@@ -0,0 +1,352 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Description": "external-shared-storage template for test_update.py::test_dynamic_file_systems_update",
```

Comment:
Minor: the description should describe what the template is going to create, not how the template is going to be used in our tests. I'd say that here the description should be something like "This template creates EBS, EFS, FSX storage that can be attached to a cluster."

Reply:
I've updated the description and will push it in my next commit